tests/test_tests.py

```diff
@@ -0,0 +1,125 @@
+# placeholder file to test the testing capability
```

Comment:
Comment is incorrect. This is an actual test file for these functions that were exposed for nttau AI optimization (though they ended up writing their own function).

Reply:
I see - I read "These are more of demonstration tests of what's implemented rather than testing the code works at a fine detail." in the top-level README.md and so thought the tests were just placeholders/demonstrations. 

Feel free to suggest a new comment.